Sources/PrimerSDK/Classes/Extensions & Utilities/UIDeviceExtension.swift

```diff
@@ -1,8 +1,8 @@
 //
-//  UIDeviceExtension.swift
+//  Strings.swift
```

Comment:
Not that it matters, but why was this renamed? Isn't there another `Strings.swift` file?

Reply:
Correct - that has been some rebase issue 👍 - fixed